src/index.ts

```diff
@@ -27,7 +27,6 @@ export {default as FlyLargeConfirm} from './components/FlyLargeConfirm';
 export {default as FlyModal} from './components/FlyModal';
 export {default as FlySelect} from './components/FlySelect';
 export {default as FlyTooltip} from './components/FlyTooltip';
-export {default as Header} from './components/Header';
```

Comment:
Be sure to re-add the new components from this PR to this file.

Reply:
Yes, this was a bad oversight 😬 Great catch!!